org.eclipse.jdt.core.manipulation/common/org/eclipse/jdt/core/manipulation/internal/javadoc/CoreJavadocAccessImpl.java

```diff
@@ -140,17 +140,13 @@ public CoreJavadocAccessImpl(IJavaElement element, Javadoc javadoc, String sourc
 		Assert.isNotNull(element);
 		Assert.isTrue(element instanceof IMember || element instanceof IPackageFragment || element instanceof ILocalVariable || element instanceof ITypeParameter);
 		fElement= element;
-		fSnippetStringEvaluator= createSnippetEvaluator(fElement);
+		fSnippetStringEvaluator= new CoreJavaDocSnippetStringEvaluator(fElement);
 		fMethod= null;
 		fJavadoc= javadoc;
 		fSource= source;
 		fJavadocLookup= JavadocLookup.NONE;
 	}
 
-	protected CoreJavaDocSnippetStringEvaluator createSnippetEvaluator(IJavaElement element) {
```

Comment:
> @robstryker Please add coments describing the usecase of such methods when you add them so they don't end up accidently removed.

please add a Junit that references the methods and consider making them public API. It's not helpfull when adopters silently rely on internals.

this method was removed because it was no longer internally referenced.

Reply:
> Btw, this change specifically doesn't have any "unused parameter" so shouldn't be part of this PR.

the parameter element is unused in both jdt ans ls. only the member fElement is used

Reply:
It's possible that it is intended to be used in jdt.ls but it hasn't been converted yet to consume it yet. 

Please remove it from the PR for now for safety. I'll try to be more careful and document in the future.